SU-SOE/stanford_soe_helper
# 7.x 2.x style tweaks

# READY FOR REVIEW

# Summary
- This has CSS changes

# Needed By (Date)
- 

# Criticality
- 

# Steps to Test
See https://stanfordits.atlassian.net/browse/SOE-2204
- Check /spotlight to verify  2 
- Check the footer to verify 3 & 4
- Does the masthead look like this in /magazine ? 
https://projects.invisionapp.com/share/WDAOBEJ4P#/screens/230196822_Section_Landing_L-f_04

# Affects 
- SoE

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2204

## Related PRs

## More Information

## Folks to notify
@josephgknox - I'd love feedback.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
